docs/UsersGuide.asciidoc

```diff
@@ -543,6 +543,23 @@ the top of the group overview page. However, it only works as operator
 or admin.
 
 
+=== Dark mode
+
+The dark mode is natively supported by most modern browsers and
+is automatically turned on when dark user interface of either
+the operating system or browser is detected. To disable the dark
+mode please Follow the instructions below:
+
+ * On Firefox, go to `about:config` and set
+   `layout.css.prefers-color-scheme.content-override`
+   to: (dark:0, light:1, system-detected:2, browser-detected:3).
```

Comment:
This can also be found in the normal appearance settings. Maybe we can just point users to about:preferences#general and that page is very self-descriptive (like on chrome).